server/src/main/java/ch/ethz/lapis/core/ConfigurationManager.java

```diff
@@ -19,14 +20,11 @@ public class ConfigurationManager {
     /**
      * Loads the configuration from both the config file and from environment variables
      */
-    public <T extends Config> T loadConfiguration(Path configPath, Class<T> configClass, String programName)
+    public LapisConfig loadConfiguration(Path configPath, String programName)
         throws IOException {
         Map<String, ?> fileConfigMaps1 = readConfigMapsFromYaml(configPath);
         Map<String, ?> envConfigMaps1 = readConfigMapFromEnvironmentVariables();
```

Comment:
@chaoran-chen Is this actually still used? Or could we also clean up the mechanism to simply read the config from the file?

Reply:
Yes, we'll remove it in a separate PR then.